app/src/main/java/com/orange/ods/app/ui/components/appbars/top/ComponentTopAppBar.kt

```diff
@@ -50,33 +48,23 @@ import com.orange.ods.compose.component.list.OdsSwitchTrailing
 import com.orange.ods.compose.text.OdsTextBody2
 import com.orange.ods.compose.text.OdsTextCaption
 import com.orange.ods.compose.theme.OdsTheme
-import kotlin.math.max
 
 @OptIn(ExperimentalMaterialApi::class)
 @Composable
 fun ComponentTopAppBar(variant: Variant) {
     val customizationState = rememberTopAppBarCustomizationState(large = remember { mutableStateOf(variant == Variant.AppBarsTopLarge) })
 
     with(customizationState) {
-        val customActionCount = max(0, actionCount.value - MainTopAppBarState.DefaultConfiguration.actions.size)
-        val customActions = NavigationItem.values()
-            .take(customActionCount)
-            .map { TopAppBarConfiguration.Action.Custom(stringResource(id = it.textResId), it.iconResId) }
-        val topAppBarConfiguration = TopAppBarConfiguration.Builder()
-            .large(isLarge)
-            .scrollBehavior(scrollBehavior.value)
-            .navigationIconEnabled(isNavigationIconEnabled)
-            .actions {
-                addAll(MainTopAppBarState.DefaultConfiguration.actions.take(actionCount.value))
-                addAll(customActions)
-                if (isOverflowMenuEnabled) add(TopAppBarConfiguration.Action.OverflowMenu)
-            }
-            .build()
+        val appBarConfiguration = AppBarConfiguration(
+            isLarge = isLarge,
+            titleRes = if (isLarge) title.value.titleResId else R.string.component_app_bars_top_regular,
```

Comment:
It seems that the `title` property of `TopAppBarCustomizationState` is only used for large top app bar. Maybe should we rename it to `largeTitle`?